activestorage/db/migrate/20190112182829_add_service_name_to_active_storage_blobs.rb

```diff
@@ -0,0 +1,5 @@
+class AddServiceNameToActiveStorageBlobs < ActiveRecord::Migration[5.2]
+  def change
+    add_column :active_storage_blobs, :service_name, :string
+  end
+end
```

Comment:
> In order to migrate existing apps users would have to re-run rails active_storage:install and rails db:migrate to add a new migration, suggestions are welcome :)

Hey, we've created approach to upgrade apps of users that use Active Storage. See https://github.com/rails/rails/pull/33419.

Could you move this migration under `activestorage/db/update_migrate/` directory so `rake app:update` will copy this migration to existing apps if needed. Also, you should extend `activestorage/db/migrate/20170806125915_create_active_storage_tables.rb` migration to add `service_name` column for new apps on `rails active_storage:install`.

Reply:
@bogdanvlviv Done, thanks for the suggestion!

What would happen if user won't execute `rake app:update` before running the app? In case he gets an error preventing him from running the app - we could also drop all [these](https://github.com/rails/rails/pull/34935/files#diff-55562e4b2a8eb648cacecb9d9d4c3ea6R67) checks. Also, [as far as I know](https://github.com/rails/rails/pull/33419#issuecomment-412327595), this task is not designed to be executed in point releases, so we should make this PR a part of 6.0.0 😅

Reply:
> What would happen if user won't execute `rake app:update` before running the app?

It is ok when an app fails because it hasn't been upgraded well. It is users' responsibility to upgrade it right, our responsibility is to guide on how to do it.

>  In case he gets an error preventing him from running the app - we could also drop all these checks. 

Yeah, I think we can drop these checks.

> Also, as far as I know, this task is not designed to be executed in point releases, so we should make this PR a part of 6.0.0 :sweat_smile:

Right, `rake app:update` is designed only fo major and minor releases. Since this PR brings a new feature, we can release it only in any next major/minor release because we [don't add features in point/patch releases](https://edgeguides.rubyonrails.org/maintenance_policy.html).

Reply:
Perfect, it made things way more simpler